src/main/kotlin/com/hedvig/underwriter/web/OnboardingSessionController.kt

```diff
@@ -100,26 +100,26 @@ class OnboardingSessionController(
 }
 
 @RestController
-@RequestMapping("/_/onboarding-sessions-se-bankid-callback")
-class OnboardingSeBankIdCallbackController(
+@RequestMapping("/_/onboarding-sessions")
+class InternalOnboardingSessionController(
     private val service: OnboardingService
 ) {
 
     companion object {
-        fun url(sessionId: UUID) = "/_/onboarding-sessions-se-bankid-callback?sessionId=$sessionId"
+        fun url(sessionId: UUID) = "/_/onboarding-sessions/$sessionId/se-bankid-callback"
     }
 
-    @PostMapping
+    @PostMapping("{id}/se-bankid-callback")
```

Comment:
I preferred the sessionId name :)
```suggestion
    @PostMapping("{sessionId/se-bankid-callback")
```

Reply:
I just like using the `id` name because it highlights that it's the primary ID of the resource presented by this controller. If I give it more custom names, to me that signals that it's some secondary ID.

These are the `OnboardingSession` controllers, so in my mind `id` is the less confusing, as I would've guessed `sessionId` refers to some query parameter foreign key or something.